.github/workflows/new_release.yml

```diff
@@ -1,36 +1,53 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI new release
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the "master" branch
-  push:
-    branches: [ "master" ]
-  pull_request:
-    branches: [ "master" ]
-
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+    inputs:
+      version:
+        description: Released new version of Realm (0.X.0+beta).
```

Comment:
why do we have `+beta` in the description

Reply:
:) removed